src/umock_c_negative_tests.c

```diff
@@ -234,4 +234,30 @@ int umock_c_negative_tests_can_call_fail(size_t index)
     return can_call_fail;
 }
 
+char* umock_c_negative_tests_get_call_description(size_t index)
+{
+    char* result = NULL;
 
+    if (umock_c_negative_tests_state != UMOCK_C_NEGATIVE_TESTS_STATE_INITIALIZED)
+    {
+        /* Codes_SRS_UMOCK_C_NEGATIVE_TESTS_31_026: [ If the module was not previously initialized, umock_c_negative_tests_can_call_fail shall return 1. ]*/
```

Comment:
>SRS_UMOCK_C_NEGATIVE_TESTS_31_026 [](http://example.com/codeflow?start=17&length=33)

The specs have been copied from the other function. This function should get specs in the .md and tests in the test suites.

Reply:
Fixed.